### ISSUE TITLE
How to create an installer ?

### ISSUE THREAD

**Issue by raamipja** [beryx-gist/badass-jlink-example-log4j2-javafx#2]
I successfully created my project with badass plugin .

I want to publish my javafx application.

How can I create a installer with this project?

**Comment by siordache** [comment]
Make sure that your JAVA_HOME points to a JDK 14 or newer distribution. Then use:
```
./gradlew jpackage
```

I updated the README with a mention about creating installable packages. I also configured a [jpackage block](https://github.com/beryx-gist/badass-jlink-example-log4j2-javafx/blob/33c403e7af539c2006e054996814bb45ff58cb60/build.gradle#L31-L35) in build.gradle to produce a nicer installer on Windows.

**Comment by raamipja** [comment]
I have a doubt in this.

I am using java jdk 14 with 64 bit. I create a jpackage with 64 bit java jdk will it work on windows 32 bit os.

will it work on both 32 bit and 64 bit windows os?

**Comment by siordache** [comment]
No, it will not work. You need to use a 32 bit JDK to create an installer for Windows 32 bit.
